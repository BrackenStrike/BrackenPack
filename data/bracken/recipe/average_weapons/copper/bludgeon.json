{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:sentry_armor_trim_smithing_template",
	"base": "minecraft:stone_sword",
	"addition": "minecraft:copper_ingot",
	"result": {
		"id": "minecraft:stone_sword",
		"components": {
			"minecraft:max_damage": 175,
			"minecraft:item_name": "Copper Bludgeon",
			"minecraft:item_model": "bracken:average_weapons/copper/bludgeon",
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "5.5 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false},
				{"translate": "10% Knockback Resistance","color":"dark_green","italic":false}
			],
			"minecraft:attribute_modifiers": [
				{
					"type": "minecraft:attack_damage",
					"id": "bracken:copper_bludgeon",
					"amount": 4.5,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:copper_bludgeon",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:knockback_resistance",
					"id": "bracken:copper_bludgeon",
					"amount": 0.1,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
			
		}
	}
}