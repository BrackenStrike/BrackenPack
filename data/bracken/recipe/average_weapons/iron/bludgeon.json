{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:sentry_armor_trim_smithing_template",
	"base": "minecraft:iron_sword",
	"addition": "minecraft:iron_ingot",
	"result": {
		"id": "minecraft:iron_sword",
		"components": {
			"minecraft:item_name": "Iron Bludgeon",
			"minecraft:item_model": "bracken:average_weapons/iron/bludgeon",
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "6 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false},
				{"translate": "10% Knockback Resistance","color":"dark_green","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:iron_bludgeon",
					"amount": 5,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:iron_bludgeon",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:knockback_resistance",
					"id": "bracken:iron_bludgeon",
					"amount": 0.1,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}