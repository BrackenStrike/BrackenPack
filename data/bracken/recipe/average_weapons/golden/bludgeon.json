{
	"type": "minecraft:crafting_shaped",
	"category": "equipment",
	"pattern": [
		" # ",
		" ##",
		"/  "
	],
	"key": {
		"#": "minecraft:gold_ingot",
		"/": "minecraft:stick"
	},
	"result": {
		"id": "minecraft:golden_sword",
		"components": {
			"minecraft:item_name": "Golden Bludgeon",
			"minecraft:item_model": "bracken:average_weapons/golden/bludgeon",
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "4 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false},
				{"translate": "Bonus Knockback","color": "aqua","italic": false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:golden_bludgeon",
					"amount": 3,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:golden_bludgeon",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_knockback",
					"id": "bracken:golden_bludgeon",
					"amount": 1,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}